docs/src/components/Example/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { LiveProvider, LivePreview, LiveEditor } from "react-live";
+import { useStaticQuery, graphql } from "gatsby";
+import styled, { css } from "styled-components";
+import defaultTheme from "@kiwicom/orbit-components/lib/defaultTheme";
+import { ButtonLink, Stack, Tooltip } from "@kiwicom/orbit-components";
+import dracula from "prism-react-renderer/themes/dracula";
+import ChevronUp from "@kiwicom/orbit-components/lib/icons/ChevronUp";
+import ChevronDown from "@kiwicom/orbit-components/lib/icons/ChevronDown";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import * as Components from "@kiwicom/orbit-components";
+import * as Icons from "@kiwicom/orbit-components/lib/icons";
+
+import Copy from "../../images/copy.svg";
+
+interface Props {
+  exampleId: string;
+}
+
+const StyledPreviewWrapper = styled.div`
+  box-shadow: ${({ theme }) => theme.orbit.boxShadowFixed};
+  padding: 20px;
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+`;
+
+const StyledBoard = styled.div``;
+
+const Example = ({ exampleId }: Props) => {
+  const [isOpened, setOpenEditor] = React.useState(false);
+  const [isCopied, setCopied] = React.useState(false);
+
+  const { allFile } = useStaticQuery(
+    graphql`
+      query ExamplesQuery {
+        allFile(filter: { absolutePath: { regex: "/__examples__/" } }) {
+          nodes {
+            id
+            relativePath
+            fields {
+              example_id
+              example
+              scope {
+                name
+                path
+                default
+              }
+            }
+          }
+        }
+      }
+    `,
+  );
+
+  const { fields } = allFile.nodes.find(n => n.fields.example_id === exampleId);
```

Comment:
If an incorrect ID is used, it will break the whole build. Could you please add a check for that? Return a note about not having that example yet or something.